### DISCUSSION TITLE
Pimcore 11 TinyMCE Default Config truncating content

### DISCUSSION THREAD

**Discussion by mlkdev** [pimcore/pimcore#16414] [Q&A / Support] [answered]
Pimcore Version 11.0.12

We have a DataObject class we're using for Articles and other publications, and for the most part it works as expected. However, we recently encountered an instance where text is being truncated without apparent reason.

We've tested to make sure it's not an issue with the database, and we were able to insert our markup directly into the database, flushed the caches and it output as expected. However, using the Pimcore UI, and the out-of-the-box config for TinyMCE, it appears that the content is being truncated before being saved.

We also followed various guides online that instruct how to remove the character limit using the following markup:
```
{
  selector: 'textarea',
  max_chars: -1
}
```
This appears to not have any affect on storage process, and still results in truncation of content. It appears to truncate around 20,000 characters. I just tried storing a block of Lorem Ipsum with no markup, and it concatenated at exactly 20,000 characters.

Anyone have any ideas?

**Comment by mlkdev** [comment, marked as answer]
The solution is to modify your `/config/packages/html_sanitizer.yaml` file to include the following definition:

```
framework:
    html_sanitizer:
        sanitizers:
            pimcore.wysiwyg_sanitizer:
                max_input_length: -1
```

There are some things to keep in mind, available for reading in the Symfony docs:
https://symfony.com/doc/current/html_sanitizer.html#max-input-length

**Comment by fashxp** [comment]
thx for sharing. added some notes to the docs too, see https://github.com/pimcore/pimcore/pull/16451
